web/assets/auto-resize-text-area.js

```diff
@@ -0,0 +1,13 @@
+const autoResizeTextArea = () => {
+    const txtarea = document.querySelectorAll(".js-auto-resize-textarea");
+    for (let i = 0; i < txtarea.length; i++) {
+        txtarea[i].setAttribute("style", "height:" + (txtarea[i].scrollHeight) + "px;overflow-y:hidden;");
+        txtarea[i].addEventListener("input", OnInput, false);
+    }
+
+    function OnInput() {
+        this.style.height = "auto";
+        this.style.height = (this.scrollHeight) + "px";
```

Comment:
I think we might need to add something like 3px to the height? The padding on the bottom seems quite tight and I get a scroll bar

![image](https://user-images.githubusercontent.com/146390/169239403-b42799b3-01aa-4f7e-899e-f2cb1cf07495.png)

Reply:
Scroll bar shouldn't be there anymore & padding should look better too now :)